ex1.py

```diff
@@ -0,0 +1,9 @@
+#area of a circle
+import math
+def Area(r):
+    return((math.pi*r**2))
+
+if __name__ == "__main__":
+    ar = str(Area(1.1))
```

Comment:
it was specified the exercise to receive an input *from the user*:
"which accepts the radius of a circle from the user"

the given data was only an example.
use the input function as explained below the exercise to get an input from the user.

Also..
the name "ar" has no meaning, give your variable a name with a meaning.